lib/Raven/CurlHandler.php

```diff
@@ -16,7 +16,6 @@
  * @package raven
  */
 
-// TODO(dcramer): handle ca_cert
```

Comment:
There wasn\'t be such todo. This worked fine because options gets from single thread curl